bravetechnologycoop/BraveButtons
# Add naming chatbot workflow (CU-24bxdpq)

- Added Button naming chatbot flow
- Updated dependencies for security reasons
- Removed the fallback Twilio response column from the `sessions` table and no longer store this data because we weren't using it and it is not planned to be stored in the shared DB schema

## Test plan:
- Deploy to Buttons Dev
- Automated tests pass
- CSV export
   - Shows "No longer used" in the Twilio column (did this so that it doesn't break any automated reports generated from the CSV)
- Push Button 3 for a button with `unit = '_NEW'`
   - See NAMING_STARTED status in Dashboard
   - Reply with a non-empty, non-'Later' name
      - Unit goes in the DB
      - See NAMING_COMPLETED status in Dashboard
      - See Responded_at and updated_at updated in Dashboard
      - Next Button push starts the normal chatbot with that name
   - Reply with an empty name
      - Get back message saying it is invalid and to try again
      - See NAMING_STARTED still in Dashboard
      - See updated_at and responded_at not change in Dashboard
      - Trying again works
   - Reply with a '_NEW' name
      - Get back message saying it is invalid and to try again
      - See NAMING_STARTED still in Dashboard
      - See updated_at and responded_at not change in Dashboard
      - Trying again works
   - Reply with a 'later' name
      - Get message back saying they can rename next time they push the button
      - See NAMING_POSTPONED in Dashboard
      - See updated_at and responded_at change in Dashboard
      - Next Button push starts the Naming chatbot again
   - Push Button 3 for the same Button again less than the session reset threshold
      - Nothing
   - Push Button 3 for the same Button again more than the session reset threshold
      - No reminder message 
      - No fallback message
      - Starts a new Session in the Dashboard with NAMING_STARTED status in the Dashboard
- Regression:
   - Normal button press with replies
   - Urgent button
   - Normal button press without replying (reminder and fallback)
      - Doesn't store the fallback Twilio responses in the DB
